webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/comet/AsyncDeployment.scala

```diff
@@ -250,26 +266,73 @@ class AsyncDeployment extends CometActor with CometListener with Loggable {
     } else NodeSeq.Empty
   }
 
+  private def showProperties:  NodeSeq = {
+    def tooltipContent(error: String): String                                = {
+      s"<h4 class='tags-tooltip-title'>Properties error</h4><div class='tooltip-inner-content'><pre class=\"code\">${StringEscapeUtils
+          .escapeHtml4(error)}</pre></div>"
+    }
+    def nodeBtnId(node: NodeId):       String                                = {
+      s"status-node-${StringEscapeUtils.escapeHtml4(node.value)}"
+    }
+    // The "a" tag has special display in the content, we need a button that opens link in new tab
+    def nodeLinkScript(node: NodeId):  NodeSeq                               = {
+      val link  = linkUtil.nodeLink(node)
+      val btnId = nodeBtnId(node)
+      scriptLinkButton(btnId, link)
+    }
+    val allNodes = nodeFactRepo.getAll()(QueryContext.systemQC).runNow
+    val nodesErrors:                   Map[MinimalNodeFactInterface, String] = propertiesStatus.flatMap {
+      case (_, _: SuccessNodePropertyHierarchy)     => None
+      case (nodeId, f: FailedNodePropertyHierarchy) => allNodes.get(nodeId).map(_ -> f.getMessage)
+    }
+    nodesErrors.toList match {
+      case Nil    => NodeSeq.Empty
+      case errors =>
+        <li class="card border-start-0 border-end-0 border-top-0">
+          <div class="card-body">
+            <h5 class="card-title d-flex justify-content-between">
+              Node configuration errors
+              <span class="badge bg-danger">{errors.size}</span>
+            </h5>
+            <ul class="list-group">
+            {
+          NodeSeq.fromSeq(
+            errors.map {
```

Comment:
I think you must limits the number of rendered nodes here, else it won't scale and the rendering will be poor/broken/lead to performance problemps. Even with just 4, it's already a lot: 
![image](https://github.com/user-attachments/assets/d10e7f3a-eae1-4f3f-90c8-49b1dca01d81)


Perhpas: 
- keep the total number of impacted nodes in the red square
- if it's 3 or less, let it like that
- if it's more, add a sentence `only displaying the first nodes in errors:`

It won't change much in UX, because in the case where there is a lot of nodes in error, it is *extremely* likely that all the errors are linked to the same inheritance problem, and so that resolving on one will resolve everywhere. 
And in addition, even if there is two root causes for the error, the user can only solve one at a time.

Reply:
OK it seems good to me

I suggest that we improve the UX and handle specific cases for the expected UX of each section (Policy generation, Groups, Nodes, Techniques) in another PR, where we would also add the possibility to collapse each section